packages/ra-core/src/types.ts

```diff
@@ -13,9 +14,11 @@ export interface RaRecord<IdentifierType extends Identifier = Identifier>
     id: IdentifierType;
 }
 
+export type SortOrder = 'ASC' | 'DESC';
```

Comment:
This type is already defined in ra-ui-materialui (packages/ra-ui-materialui/src/field/types.ts), I'm afraid of a conflic. As this change seems to be unrelated to the subject of the PR, could you revert it?